service/templates/display_title.html

```diff
@@ -174,7 +174,7 @@ <h5 class="expand-bottom-half {% if loop.index != 1 %} expand-top-half {% endif
   <div class="text printhidden">
     <h6 class="heading-medium">{{ _('What next') }}</h6>
     <ul>
-      <li><a href="/drv-15/search">{{ _('Search for another title using this service') }}</a></li>
+      <li><a href="{{ url_for('find_titles') }}">{{ _('Search for another title using this service') }}</a></li>
```

Comment:
If I remember rightly the 'find_titles' link is a bit ambiguous - the behaviour varies according to whether the action is Get or Post. This usage is probably OK however.